cfme/tests/cloud_infra_common/test_relationships.py

```diff
@@ -139,7 +143,7 @@ def wait_for_relationship_refresh(provider):
 
 @pytest.mark.parametrize("relationship,view", HOST_RELATIONSHIPS,
     ids=[rel[0] for rel in HOST_RELATIONSHIPS])
-@pytest.mark.provider([VMwareProvider], selector=ONE_PER_TYPE)
+@pytest.mark.provider([InfraProvider])
```

Comment:
**REQUIRED:** I think we still want the ONE_PER_TYPE selector in there right? We only want ONE vmware, and ONE rhev etc....

Reply:
Okay